src_common/common_utils.c

```diff
@@ -214,50 +214,25 @@ bool amountToString(const uint8_t *amount,
     return true;
 }
 
-bool getEthAddressFromKey(cx_ecfp_public_key_t *publicKey, uint8_t *out, cx_sha3_t *sha3Context) {
+void getEthAddressFromRawKey(const uint8_t raw_pubkey[static 65],
```

Comment:
I think I prefer the old version tbh
Plus this is used by plugins and we don't really want to break the compatibility

Reply:
Is it directly used by the plugins, or indirectly from the source files from src_plugins/xxx? In which case, there is issue??

Reply:
At least Opensea
https://github.com/LedgerHQ/app-plugin-opensea/blob/c64c989e4965c9654f43208123a949bbc4b5e922/src/utils.c#L22

Reply:
But it's archived 🤔

Reply:
Just in case, can you leave old implem behind with deprecate flag ?

Reply:
Would be nice to be able to build all deployed plugins against a specific ethereum SDK ref.
To be able to easily test if a change will create deployments issues.